### PR TITLE
dependencies: restrict max version of wtforms

### LINKED ISSUE #228: pin wtforms >2,<3 in invenio-oauth2server (and release)

### PR DESCRIPTION
**Pull request by ppanero**
closes #228 

includes release commit and changelog

CI has been broken for the last 5 months... installation takes forever locally (is been resolving for the las 40')

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,12 +59,13 @@
...
     'invenio-accounts>=1.3.1',
-    'invenio-base>=1.2.3',
+    'invenio-base>=1.2.4',
     'invenio-i18n>=1.2.0',
     'invenio-theme>=1.3.4',
     'pyjwt>=1.5.0',
     'requests-oauthlib>=1.1.0,<1.2.0',
     'WTForms-Alchemy>=0.15.0',
+    'WTForms>=2.0.0,<3.0.0',
 ]
```
Comment (fenekku) [inline setup.py:69]: The invenio-db constraints also needs to be updated to: `'>=1.0.9,<2.0.0'` for mysql, postgresl, sqlite.

**Review by fenekku — COMMENTED** [review]
For the requirements taking forever, we may have to add `'flask>=1.1,<2'` :disappointed:

**Review by fenekku — APPROVED** [review]
Yup that's good to go :shipit: !

File: tests/conftest.py [hunk 2]
```diff
@@ -71,6 +71,19 @@ def init_app(app):
             SQLALCHEMY_TRACK_MODIFICATIONS=True,
             TESTING=True,
             WTF_CSRF_ENABLED=False,
+            APP_THEME="semantic-ui",
```
Comment (ntarocco) [inline tests/conftest.py:74]: This should not be needed, we have fixed it in `pytest-invenio`:
https://github.com/inveniosoftware/pytest-invenio/commit/15c15bd60b042d989bc4c52d5c9411a5e7b73919

can you try to bump the `pytest-invenio>=1.4.2`?

Comment (jrcastro2) [reply tests/conftest.py:74]: Doesn't seem to work, bumped here <https://github.com/jrcastro2/invenio-oauth2server/pull/1>, but not working.

Comment (ntarocco) [reply tests/conftest.py:74]: As discussed, the fixtures are not using `pytest-invenio` ones. It requires too many changes at the moment.

**Review by ntarocco — APPROVED** [review]

**Pull request merged**